ui/tests/acceptance/enterprise-namespaces-test.js

```diff
@@ -59,8 +59,10 @@ module('Acceptance | Enterprise | namespaces', function (hooks) {
     await authPage.tokenInput('root').submit();
     await settled();
     await click('[data-test-namespace-toggle]');
-    await waitFor('[data-test-current-namespace]');
-    assert.dom('[data-test-current-namespace]').hasText('/beep/boop/', 'current namespace begins with a /');
```

Comment:
seeing this i wonder if the old namespace would send users to `/namespace` and that's one reason why we had to account for that initial forward slash? 🤔